auth-web/src/views/auth/AcceptInviteView.vue

```diff
@@ -97,6 +92,9 @@ export default class AcceptInviteView extends Mixins(NextPageMixin) {
             user: null
           }
           this.setCurrentMembership(membership)
+          // Store the invitation org as acuurentaccount in session for handling multiple accounts in a single IDIR.
+          // Preserves the invitation org while page refresh.
+          ConfigHelper.addToSession(SessionStorageKeys.CurrentAccount, JSON.stringify({ id: invitation.membership[0].org.id, label: invitation.membership[0].org.name }))
```

Comment:
If user moves out of the flow, what will happen?

Reply:
Removed this approach, now we add it as a variable in child and append it on save